bundles/org.openhab.binding.oppo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -205,13 +221,21 @@
 		<item-type>String</item-type>
 		<label>Disc Type</label>
 		<description>The Current Type of Disc in the Player</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>MediaControl</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="audio_type">
 		<item-type>String</item-type>
 		<label>Audio Type</label>
 		<description>The Current Audio Track Type</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>SoundVolume</tag>
```

Comment:
This doesn't seem like sound volume based on the item type/label/description?

Reply:
Changed to Status+Info